client/src/components/Header/Header.css

```diff
@@ -0,0 +1,38 @@
+.Foot {
+    position: fixed;
+    top: 0%;
+    left: 0%;
+    width: 100%;
+    background-image: linear-gradient(to right, #1869a7, #80800c1b );
+    border-bottom-left-radius: 30px;
+    border-bottom-right-radius: 30px;
+    box-shadow: 10px 5px 10px 1px rgba(0, 0, 0, 0.373) ;
+    transition: all 0.5s ease-out 100ms;
+}
+
+.Foot:hover {
+    box-shadow: 10px 5px rgba(210, 211, 225, 0.373) ;
+}
+
+.FootContainer {
+    padding-left: 40px;
+    padding-right: 40px;
+    height: auto;
```

Comment:
Would recommend setting this to a set value (ex: 120px) to prevent subpages from colliding with the header